docs/reference/config-params.md

```diff
@@ -5,196 +5,227 @@ title: Kev configuration reference
 
 # Configuration
 
-Kev leverages Docker Compose specification to configure and prepare an application for deployment in Kubernetes. Environment configuration lives in a dedicated docker compose override file, which automatically gets applied to the project's source sdocker compose files at `render` phase.
+Kev leverages the Docker Compose specification to configure and prepare an application for deployment in Kubernetes.
+
+## Project wide configuration
+
+This is optional, but if provided, lives in the base docker compose files (or at least one of the files) used to initialise the project.
```

Comment:
tiny reword suggestion: `Project wide configuration is OPTIONAL and can be defined in one (or more) of the source compose files used to initialise the project.` ... followed with `It will be applied against ..`